examples/jsm/postprocessing/Pass.js

```diff
@@ -39,13 +39,19 @@ class Pass {
 
 // Helper for passes that need to fill the viewport with a single quad.
 
-const _camera = new OrthographicCamera( - 1, 1, 1, - 1, 0, 1 );
+const _camera = /* @__PURE__ */ new OrthographicCamera( - 1, 1, 1, - 1, 0, 1 );
 
 // https://github.com/mrdoob/three.js/pull/21358
 
-const _geometry = new BufferGeometry();
-_geometry.setAttribute( 'position', new Float32BufferAttribute( [ - 1, 3, 0, - 1, - 1, 0, 3, - 1, 0 ], 3 ) );
-_geometry.setAttribute( 'uv', new Float32BufferAttribute( [ 0, 2, 0, 0, 2, 0 ], 2 ) );
+const _geometry = /* @__PURE__ */ ( () => {
+
+	const _geometry = new BufferGeometry();
+	_geometry.setAttribute( 'position', new Float32BufferAttribute( [ - 1, 3, 0, - 1, - 1, 0, 3, - 1, 0 ], 3 ) );
+	_geometry.setAttribute( 'uv', new Float32BufferAttribute( [ 0, 2, 0, 0, 2, 0 ], 2 ) );
+
+	return _geometry;
+
+} )();
```

Comment:
I can refactor this into an internal `FullscreenTriangleGeometry` class here or in another PR if that's better understood.

```js
// https://github.com/mrdoob/three.js/pull/21358

class FullscreenTriangleGeometry extends BufferGeometry {

  constructor() {

    super();

    this.setAttribute( 'position', new Float32BufferAttribute( [ - 1, 3, 0, - 1, - 1, 0, 3, - 1, 0 ], 3 ) );
    this.setAttribute( 'uv', new Float32BufferAttribute( [ 0, 2, 0, 0, 2, 0 ], 2 ) );

  }

}

const _geometry = /* @__PURE__ */ new FullscreenTriangleGeometry();
```

For code golfing, I also have a far worse version you can do on the GPU since #24179.

```glsl
vUv = vec2(gl_VertexID << 1 & 2, gl_VertexID & 2);
gl_Position = vec4(vUv * 2.0 - 1.0, 0, 1);
```